kubevirt/k6t-only-instancetype/k6t-require-instancetype.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: k6t-require-instancetype
+  annotations:
+    policies.kyverno.io/title: Require instanceType
+    policies.kyverno.io/category: KubeVirt
+    kyverno.io/kyverno-version: 1.6.2
```

Comment:
This annotation means you tested and verified it on 1.6.2. Is that correct?

Reply:
*cogh* I was optimistic, I'm still trying to run minikube on Silverblue in order to verify it.